### ISSUE TITLE
slideNext and slidePrev jump to slide

### ISSUE THREAD

**Issue by gfnool** [nolimits4web/Swiper#1222]
Hi,

this is a bit difficult to catch but I have found that sometimes if you click the next and prev button the swiper jump straight to the (prev slide -1) or (next slide + 1) without showing the slide animation. Then clicking again on the button doesn't move the slider because the active slide is yet there.
It happens when you click the button near the end of the transition. It's a bit tricky to get but even the demo shows this bug:
http://www.idangero.us/swiper/demos/14-nav-arrows.html

**Comment by nolimits4web** [comment]
Really, i can't catch it. What is your device/os? Related to this http://www.idangero.us/swiper/forum/#!/bugs-and-issues:nextbuttonprevbutton-doesn ?

**Comment by gfnool** [comment]
Yes, it's difficult to catch it. Here there's a video in slow motion:
https://www.youtube.com/watch?v=ABmxFkGwx44
I've test it in Chrome and Firefox on OSX.

**Comment by nolimits4web** [comment]
Thanks for the video, but man, i still can't replicate it, also on Chrome/FF/Safari on OSX

**Comment by cspicuzza** [comment]
I'm having the same exact issue but with the pagination.  I'm using the coverflow effect; if you drag the slide you can see it transforming.  If you click the pagination it just skips to the slide without animation.   It just recently started this in one of the latest updates.

I'm currently using the jquery version 3.07.  This has been replicated in Chrome/FF

**Comment by cspicuzza** [comment]
I just wanted to follow up.  I was checking out the demo gfnool posted.  I saw it's using swiper.min.js instead of jquery.swiper.min.js.  I switched to swiper.min.js on my project and now the animations are working.  

TLDR:  the bug is in jquery.swiper.min.js

**Comment by webmatrixxxl** [comment]
H mange to have same problem OS X 10.4 Chrome Version 44.0.2403.125 (64-bit)
Here is a video:
http://quick.as/56wdh5bpw

**Comment by baptistebriel** [comment]
Same issue here using swiper.min.js. Sometimes, the animation breaks. The current slide disappears and the prev (or) next slide shows up. It looks like it doesn't happen when swiping, but just when using the arrow buttons.

**Comment by nolimits4web** [comment]
It is issue in Chrome with rendering, nothing can do with it at the moment

**Comment by gfnool** [comment]
Just to remember that the bug affected also Firefox as mentioned in the 2nd post.

**Comment by webmatrixxxl** [comment]
There is no problem in FireFox 40.0 on OSX 10.4.4

**Comment by webmatrixxxl** [comment]
@nolimits4web changing slides on Chrome Version 44.0.2403.157 (64-bit) OSX 10.10.5 is terrible. Still skip the animation if changing slides quick. Can you fix this ?

**Comment by burbanczyk** [comment]
Having the exact same issue in chrome under windows. FF looks fine

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with issue
